src/core/jsonl/jsonl.cc

```diff
@@ -1,3 +1,5 @@
+#include "sourcemeta/core/json_value.h"
+#include <istream>
```

Comment:
Same here. Please put system includes AFTER sourcemeta includes with a blank line in between, and with a comment noting down why its being included.